.github/workflows/ci-core.yml

```diff
@@ -120,6 +120,15 @@ jobs:
           certManager:
             enabled: ${{ matrix.enableCertManager }}
             generateCA: true
+        extraObjects:
+          - apiVersion: keda.k8s.io/v1alpha1
+            kind: ClusterTriggerAuthentication
+            metadata:
+              name: aws-credentials
+              namespace: keda
+            spec:
+              podIdentity:
+                provider: aws-eks
```

Comment:
I think that here you have a race condition because `ClusterTriggerAuthentication` isn't created before trying to render the extra object. Maybe using any kind of hook to delay the process you can achieve this